### PR TITLE
chore: deploy to association environment

### PR DESCRIPTION
**Pull request by evegufy**
## Description

deploy to association environment

## Why

https://github.com/eclipse-tractusx/portal/issues/349

### CODE REVIEW

File: environments/argocd-app-templates/appsetup-int.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+metadata:
+  name: dim-wrapper
+spec:
+  destination:
+    namespace: product-portal
+    server: 'https://kubernetes.default.svc'
+  source:
+    path: charts/dim
+    repoURL: 'https://github.com/phil91/dim-client.git'
+    targetRevision: chore/349-deploy-to-eV
```
Comment (Phil91) [inline environments/argocd-app-templates/appsetup-int.yaml:31]: is this planned to stay like this? or are we changing it to an tag?

File: environments/helm-values/values-int.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    - host: "dim.int.catena-x.net"
+      paths:
+        - path: "/api/dim"
+          pathType: "Prefix"
+          backend:
+            port: 8080
+
+dim:
+  swaggerEnabled: true
+  rootDirectoryId: "27fee02a-e265-4cfc-af70-4f217a33840b"
```
Comment (Phil91) [inline environments/helm-values/values-int.yaml]: this should be changed to the sap provided value for the newly created directory. maybe change it to tbd for now?

Comment (evegufy) [reply environments/helm-values/values-int.yaml]: @Phil91 thanks for clarifying! will do, same for operatorId below, right?

File: environments/argocd-app-templates/appsetup-int.yaml (hunk 1 above)
Comment (evegufy) [reply environments/argocd-app-templates/appsetup-int.yaml:31]: yes, will be changed with release of the next version

**Pull request merged**